pom.xml

```diff
@@ -53,6 +53,11 @@
           </exclusion>
         </exclusions>
       </dependency>
+      <dependency>
+          <groupId>org.jenkins-ci.plugins</groupId>
+          <artifactId>structs</artifactId>
+          <version>1.3</version>
```

Comment:
Changed to 1.3 for testing.
(1.2 causes `java.lang.NoClassDefFoundError: org/jenkinsci/plugins/structs/describable/UninstantiatedDescribable` in tests)

Reply:
Use a newer parent POM and you will see such mistakes in the `validate` phase, with a more meaningful error message.